tests/cfncluster-release-check.py

```diff
@@ -19,10 +19,13 @@
 # against limits in each region, the number of simultaneously built
 # clusters in each region is a configuration parameter.
 #
-# NOTE: To simplify this script, at least one subnet in every region
-# to be tested must have a resource tag named "CfnClusterTestSubnet"
-# (value does not matter).  That subnet will be used as the launch
-# target for the cluster.
+# NOTE:
+# - This script requires python2
```

Comment:
so it doesn't work with 3? What's the effort to let the script works also with 3?

Reply:
The scope of this PR should be just fix the script it to make it work.
We can upgrade the code to work with python3 by changing the _Queue_ and the _iteritems()_ stuff, but I think we can make it in a different patch.